daxgen/daxgen.py

```diff
@@ -0,0 +1,184 @@
+import networkx as nx
```

Comment:
It is third party library/package allowing you to deal with graphs easily. See their [website](https://networkx.github.io/) for details if interested.